srcpkgs/gcovr/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'gcovr'
 pkgname=gcovr
 version=4.2
-revision=3
+revision=4
 build_style=python3-module
-pycompile_module="gcovr"
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-Jinja2"
+depends="python3-setuptools python3-Jinja2 python3-lxml"
+checkdepends="$depends python3-pytest"
```

Comment:
And probably all these: https://github.com/gcovr/gcovr/blob/master/requirements.txt to checkdepends.